PeerioTechnologies/peerio-mobile
# chore: update file migration dialog copy [branch ch8083]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/8083/migration-dialog-update-2
#### Testing instructions  
Verify that migration copy has changed and that the initial message for the first few seconds is always "This dialog will close when the update is complete". Also, when the initial message is showing, "Did you know ...?" is not shown.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
